packages/tests-e2e/plugins/inner-blocks-templates/index.js

```diff
@@ -72,4 +73,44 @@
 
 		save,
 	} );
+
+	registerBlockType( 'test/test-inner-blocks-transformer-target', {
+		title: 'Test Inner Blocks transformer target',
+		icon: 'cart',
+		category: 'common',
+
+		transforms: {
+			from: [
+				{
+					type: 'block',
+					blocks: ['test/i-dont-exist', 'test/test-inner-blocks-no-locking', 'test/test-inner-blocks-locking-all', 'test/test-inner-blocks-paragraph-placeholder'],
```

Comment:
Nitpick: This does not adhere to style standards. There should be a space after `[` and before `]`. There's another instance below as well in the `to` definition.